packages/react-examples/src/react-charting/LineChart/LineChart.CustomAccessibility.Example.tsx

```diff
@@ -226,6 +226,7 @@ export class LineChartCustomAccessibilityExample extends React.Component<
           colorFillBars={colorFillBarData}
           allowMultipleShapesForPoints={this.state.allowMultipleShapes}
           enablePerfOptimization={true}
+          enableReflow={true}
```

Comment:
Doesn't linechart has enableReflow = true by default

Reply:
I added it for better clarity. Should I remove it from line chart examples?

Reply:
Yes we can remove.

Reply:
done